packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -11,6 +11,7 @@ def moduleProjectRoot = project(':flutter').projectDir.parentFile.parentFile
 
 // Note: if this logic is changed, also change the logic in app_plugin_loader.gradle.
 def pluginsFile = new File(moduleProjectRoot, '.flutter-plugins-dependencies')
+List<String> pluginsNames = new ArrayList<>()
```

Comment:
```suggestion
```

Reply:
this isn't resolved yet. Did you try using the existing list?

Reply:
It was resolved by the last commit. It has been changed to use `object.plugins.android.name`.
[changed file](https://github.com/chunfengyao/flutter/blob/chunfengyao-patch-for-issue-92252/packages/flutter_tools/gradle/module_plugin_loader.gradle)